docs/apidocs/openapi-spec/openapiv2.yaml

```diff
@@ -2420,7 +2420,7 @@ definitions:
         type: string
       forkBranchPatterns:
         type: string
-      gitPrivate:
+      gitPublic:
```

Comment:
Is this change backward compatible?

Reply:
On the CRD level yes, thanks to a custom UnmarshallJSON - https://github.com/jenkins-x/jx/pull/5284/commits/298486187de88b7b614856838c7710f0708d5a88#diff-2f0b17514b6caac80feb96e8fa27a5a5R428